client/rest-high-level/src/main/java/org/elasticsearch/client/dataframe/transforms/DataFrameTransformConfig.java

```diff
@@ -170,4 +170,46 @@ public int hashCode() {
     public String toString() {
         return Strings.toString(this, true, true);
     }
+
+    public static Builder builder() {
```

Comment:
Do you think it would make sense to make DataFrameTransformConfig constructor private now that we have "builder()" as a recommended entry point to this class? The same question for other classes...

Reply:
That is a good thought, How about "package private" so that tests can still utilize it?

Reply:
Package-private SGTM